pkg/util/testingjobs/jobset/wrappers.go

```diff
@@ -81,15 +83,20 @@ func (j *JobSetWrapper) Suspend(s bool) *JobSetWrapper {
 	return j
 }
 
-// Queue updates the queue name of the JobSet.
-func (j *JobSetWrapper) Queue(queue string) *JobSetWrapper {
+// Label sets a label to the JobSet.
+func (j *JobSetWrapper) Label(k, v string) *JobSetWrapper {
```

Comment:
Are these changes in use in any tests, or is it WIP to enable adding integration tests?

I would like to have at least a basaline scenario integration test for the prebuilt-workload, similar to the Job.

We care about JobSet, because this is the first Job CRD that will be fully usable in MultiKueue.

Reply:
used in `pkg/controller/jobs/jobset/jobset_webhook_test.go`

> I would like to have at least a basaline scenario integration test for the prebuilt-workload, similar to the Job.

The prebuilt workload support is mainly implemented in the jobfamwork and validated in the batch/Job integration tests. The bits that are specific to Jobset are tested in unit tests.

Reply:
I don't see how the unit tests show correctness of the code changes scattered in webhooks.

Reply:
Both the changes and the tests are in the webhooks.

Reply:
Sure, but I think we need some higher level test (maybe e2e) to prove the solution works as a whole. Technically, every PR could be covered with just unit tests, but for changes scattered in various compontents it is good to have integration tests. The integration tests have also the documenting role. They show *why* changes in individual files are needed.